include/umf/providers/provider_os_memory.h

```diff
@@ -70,6 +70,8 @@ typedef struct umf_os_memory_provider_params_t {
 
     /// Describes how node list is interpreted
     umf_numa_mode_t numa_mode;
+    /// part size for interleave mode - 0 means default (system specific)
+    size_t part_size;
```

Comment:
Does part_size need to be muliple of page size? If so, perhaps we should document that and check during OS provider init?

Reply:
it do not need